packages/nextjs/src/config/loaders/wrappingLoader.ts

```diff
@@ -202,8 +202,13 @@ export default function wrappingLoader(
       templateCode = templateCode.replace(/__COMPONENT_TYPE__/g, 'Unknown');
     }
 
-    if (sentryConfigFilePath) {
-      templateCode = `import "${sentryConfigFilePath}";`.concat(templateCode);
+    if (sentryConfigFilePath && path.isAbsolute(this.resourcePath)) {
+      const sentryConfigFileImportPath = path
+        .relative(this.resourcePath, sentryConfigFilePath) // Get path relative to current module because webpack can't handle absolute paths on windows: https://github.com/getsentry/sentry-javascript/issues/8133
+        .replace(/\.[^/.]+$/, ''); // Remove the file extension from the import
```

Comment:
You also need to replace every `\` with `/`. Import paths cannot use backslashes.

Reply:
It's worth noting that in my testing, the file extension didn't seem to matter

Reply:
Ah thanks for pointing that out. Was just messing around here and yeah doesn't seem to fix anything haha